broskichat-app/src/App.jsx

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from 'react';
 
 import './App.css';
+
 // Imports of ./Components
 import Header from './Components/Partials/Header';
 import Home from './Components/Home/Home';
 import Footer from './Components/Partials/Footer';
+import About from './Components/About/About';
 
 class App extends Component {
   render() {
 
     return (
+
       <React.Fragment>
         <Header />
+
```

Comment:
If these extra spaces make it easier to understand what's going on we should keep them, otherwise let's remove.